config/locales/kinetic_cafe_error.fr-CA.yml

```diff
@@ -12,3 +12,5 @@ fr-CA:
       header:
         status: Condition
         code: Code
+    cause: >-
+      ^-- causé par: %{message}
```

Comment:
This will be addressed in a future version which will improve what KC Errors does with automatic logging. (To make this work, we actually need to be able to pass in the desired locale for both `#message` and `#i18n_message`, which is not currently possible.)